src/main/webapp/app/programming/manage/code-editor/instructor-and-editor-container/code-editor-instructor-and-editor-container.component.html

```diff
@@ -102,15 +102,21 @@
             <!-- action buttons -->
             <div>
                 @if (!exercise.studentParticipations || !exercise.studentParticipations.length) {
-                    <button (click)="createAssignmentParticipation()" [disabled]="loadingState !== LOADING_STATE.CLEAR" class="btn btn-primary me-2">
-                        @if (loadingState !== LOADING_STATE.CREATING_ASSIGNMENT_REPO) {
-                            <fa-icon [icon]="faPlus" class="text-white me-2" />
-                        }
-                        @if (loadingState === LOADING_STATE.CREATING_ASSIGNMENT_REPO) {
-                            <fa-icon [icon]="faCircleNotch" animation="spin" class="text-white me-2" />
-                        }
-                        <span jhiTranslate="artemisApp.editor.repoSelect.createAssignmentRepo"></span>
-                    </button>
+                    <div
+                        [ngbTooltip]="'artemisApp.editor.repoSelect.createAssignmentRepoDisabledTooltip' | artemisTranslate"
+                        [disableTooltip]="!isCreateAssignmentRepoDisabled"
+                        class="d-inline-block"
+                    >
+                        <button (click)="createAssignmentParticipation()" [disabled]="isCreateAssignmentRepoDisabled" class="btn btn-primary me-2">
+                            @if (loadingState !== LOADING_STATE.CREATING_ASSIGNMENT_REPO) {
+                                <fa-icon [icon]="faPlus" class="text-white me-2" />
+                            }
+                            @if (loadingState === LOADING_STATE.CREATING_ASSIGNMENT_REPO) {
+                                <fa-icon [icon]="faCircleNotch" animation="spin" class="text-white me-2" />
+                            }
```

Comment:
Please use @else instead 

```suggestion
                            @if (loadingState !== LOADING_STATE.CREATING_ASSIGNMENT_REPO) {
                                <fa-icon [icon]="faPlus" class="text-white me-2" />
                            } @else {
                                <fa-icon [icon]="faCircleNotch" animation="spin" class="text-white me-2" />
                            }
```